### ISSUE TITLE
AttributeError on document insertion

### ISSUE THREAD

**Issue by johnymontana** [neo4j-contrib/neo4j_doc_manager#43]
When importing documents mongo connector crashed with this error:

~~~
mongo-connector -v -m localhost:27017 -t http://localhost:7474/db/data -n catalog.course,catalog.category,catalog.university,catalog.instructor,catalog.student,catalog.course_taken -d neo4j_doc_manager
Logging to mongo-connector.log.
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/Library/Python/2.7/site-packages/mongo_connector/util.py", line 85, in wrapped
    func(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/mongo_connector/oplog_manager.py", line 256, in run
    docman.upsert(doc, ns, timestamp)
  File "/Users/lyonwj/neotechnology/devtmp/forks/neo4j_doc_manager/mongo_connector/doc_managers/neo4j_doc_manager.py", line 60, in upsert
    builder = NodesAndRelationshipsBuilder(doc, doc_type, doc_id)
  File "/Users/lyonwj/neotechnology/devtmp/forks/neo4j_doc_manager/mongo_connector/doc_managers/nodes_and_relationships_builder.py", line 17, in __init__
    self.build_nodes_query(doc_type, doc, doc_id)
  File "/Users/lyonwj/neotechnology/devtmp/forks/neo4j_doc_manager/mongo_connector/doc_managers/nodes_and_relationships_builder.py", line 30, in build_nodes_query
    self.build_nodes_query(key, document[key], id)
  File "/Users/lyonwj/neotechnology/devtmp/forks/neo4j_doc_manager/mongo_connector/doc_managers/nodes_and_relationships_builder.py", line 35, in build_nodes_query
    self.build_nodes_query(json_key, json, id)
  File "/Users/lyonwj/neotechnology/devtmp/forks/neo4j_doc_manager/mongo_connector/doc_managers/nodes_and_relationships_builder.py", line 22, in build_nodes_query
    for key in document.keys():
AttributeError: 'NoneType' object has no attribute 'keys'
~~~

Steps to reproduce:
* with neo4j doc manager running, import this [collection dump](https://dl.dropboxusercontent.com/u/67572426/course.json):
~~~
mongoimport -d catalog -c course --host=127.0.01 < course.json
~~~

**Comment by hannelita** [comment]
Please refer to https://github.com/neo4j-contrib/neo4j_doc_manager/pull/45

**Comment by johnymontana** [comment]
Still getting same error as above when importing the referenced [JSON file](https://dl.dropboxusercontent.com/u/67572426/course.json)

~~~
mongoimport -d catalog -c course --host=127.0.01 < course.json
~~~

**Comment by johnymontana** [comment]
Fixed in PR #45
